Documentation/ceph-nfs-crd.md

```diff
@@ -25,6 +25,8 @@ metadata:
   name: my-nfs
   namespace: rook-ceph
 spec:
+  # rados property is not used in versions of Ceph equal to or greater than 
+  # 16.2.6, see note in RADOS settings section below.
```

Comment:
There are a few places like this in the docs/comments that still need to be changed from .6 to .7